apps/website/src/pages/docs/tooltip.mdx

```diff
@@ -21,9 +21,9 @@ A tooltip is a small container that typically appears above the element you are
 
 ## Usage
 
-You should use up to 256 characters. Content other than text such as pictures, tables, etc, are not recommended. Mobile devices does not have a hover state, caution when using tooltips in desktop apps that have a mobile counterpart. There is no determined way of replacing tooltips in mobile devices.
+You should use up to 256 characters. Content other than text such as pictures, tables, etc., are not recommended. Mobile devices do not have a hover state. Thus, be cautious when using tooltips in desktop apps that have a mobile counterpart. Since there is no determined way of replacing tooltips on mobile devices.
```

Comment:
The beginning of the last sentence is a bit confusing giving the rest of the text.

Reply:
Changed the full stop before the last sentence to a comma, to make the message more clearer.

Reply:
LGTM if you want to merge